csrc/ops/composite.cpp

```diff
@@ -89,6 +89,40 @@ TensorView* matmul(TensorView* a, TensorView* b) {
   return matmul(a, b, true /* cast output to input dtype */);
 }
 
+TensorView* linear(TensorView* a, TensorView* b, TensorView* bias) {
+  NVF_CHECK(
+      a->nDims() == b->nDims(),
+      "The number of dimension of A and B do not match");
+  NVF_CHECK(a->nDims() == 2, "Only 2-D A/B Tensors are supported in Linear!");
```

Comment:
similarly, can we add a TODO here as well to expand the support?

Reply:
I went based on the API (as you saw as well). I can a TODO here.

Reply:
>Not against @kevinstephano 's suggestion on keeping it as-is for faster movement. @jacobhinkle 's suggestion doesn't seem to suggest lot more work... We should at least follow up with that as a clean up.

 I don't mind doing - superficially it's not a lot of work - but this gets stuck in SerDe. We can chat about it.

Reply:
Let's move forward with 2 C++ functions for `linear` and get the PR merged!